exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/join/TestHashJoinAdvanced.java

```diff
@@ -160,4 +165,19 @@ public void testJoinWithMapAndDotField() throws Exception {
       .baselineValues("1", "2", "1", null, "a")
       .go();
   }
+
+  @Test
+  public void testHashLeftJoinWithEmptyTable() throws Exception {
+    TestNestedLoopJoin.testWithEmptyJoin(dirTestWatcher.getRootDir(), "left outer", hjpattern, 1155L);
```

Comment:
It is not so good to use helper methods from `TestNestedLoopJoin` in `TestHashJoinAdvanced` class. 
Could you factor out the common part of code (helper methods and constants) into the Base class, for example `JoinTestBase`?